gcp/api/Pipfile

```diff
@@ -15,3 +15,4 @@ grpcio-reflection = "*"
 
 [dev-packages]
 grpcio-tools = "*"
+yapf = "*"
```

Comment:
Is this needed when the root `Pipfile` also contains this? Introducing this here could also introduce conflicting versions and behaviour.

Reply:
I think it's needed? Root pipfile is not applied here since pipenv does not actually read other Pipfile's apart from the current one even if you import it.

Reply:
Hmm, the way I typically do development (and as recommended in https://github.com/google/osv.dev/blob/master/CONTRIBUTING.md#prerequisites) 

Is to do a `pipenv shell`  at the root for all development. That gives us all the needed dev tools in a consistent way. This is how the github actions does things.
 
Then to run stuff/tests etc in e.g. gcp/api, we just do `pipenv run` which uses the Pipfile in those directories.

Reply:
That makes sense, removed the line.